### PR TITLE
Input box for location: something else

### PR DESCRIPTION
**Pull request by sakura-sketch**
Added an extra text field for location "Something else"

### CODE REVIEW

**Comment by dsamojlenko** [comment]
One minor issue. Seems the `category_description` field still gets populated if you select "Something else", fill out the field, then select a different category.

To recreate:
- on the Category form, select "Something else"
- Put something in the category description field
- Select any other category (because I changed my mind)
- Continue the process to generate a poster
- Check the database - the category_description has a value

**Comment by YedidaZalik** [comment]
English looks good.
French translation for text box for Something else
Please assist @EliseKa 
Merci!

File: locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -2036,6 +2037,12 @@ msgstr "Espaces religieux"
 msgid "Something else"
 msgstr "Autre"
 
+#: register/forms.py
+#, fuzzy
+#| msgid "Address of service or event"
+msgid "Give brief description of service or event"
+msgstr "Adresse du service ou de l’événement"
+
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:2045]: fuzzy match

**Comment by EliseKa** [comment]
I just added the translation for the text box, but looks like one of the checks didn't pass

File: register/static/register/js/something-else.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ for (var i = 0, len = radios.length; i < len; i++) {
     radios[i].onclick = function() {
       if (this.checked && this.value === "other") {
         document.getElementById("id_category-category_description").parentElement.style.display = 'block';
+        document.getElementById("id_category-category_description").value = "";
```
Comment (dsamojlenko) [inline register/static/register/js/something-else.js:15]: This didn't quite fix what I had identified before. You still have this issue:
- Fill out the form
- Select "Something Else"
- Enter some text in category_description
- Click next
- Click back
- Select a different category
- Continue to the end
- Check database, `category_description` is populated.

**Review by dsamojlenko — APPROVED** [review]

**Pull request merged**